internal/ignition/templates/discovery.ign

```diff
@@ -176,6 +181,24 @@
         },
         "contents": { "source": "data:text/plain;base64,{{.PreNetworkConfigScript}}"}
     }{{end}}{{range .StaticNetworkConfig}},
+    {
+      "path": "{{.FilePath}}",
```

Comment:
Is this duplicated starting on line 201?

Reply:
I've separated the current flow from the new implementation, so when we're ready to fully transition to the new approach, it will be easy to remove the related parts.

Reply:
Okay I think the diff is just a bit strange. I guess only one of `StaticNetworkConfig` or `StaticNetworkConfigWithNmstatectl` will be defined, right?

Reply:
correct, but since I created a new file named `pre-network-manager-config-with-nmstatectl.service`, I need distinct variables for each file to differentiate between them.